### PR TITLE
docs: add warning re:defer to mhchem documentation

### LINKED ISSUE #2482: mhchem.js will not run if katex.js is set to "defer"
JS Bin with .min.js version: https://jsbin.com/runowevafi/edit?html,output
JS Bin with .js version: https://jsbin.com/bagafidifi/1/edit?html,output

Both show the same error: Cannot read property '__defineMacro' of undefined

(After the change, you will need to fix the integrity hash at <https://github.com/KaTeX/KaTeX/blob/master/contrib/mhchem/README.md>. While you are at it, please add the missing `crossorigin="anonymous"`.)

### PR DESCRIPTION
**Pull request by ronkok**
Fix #2482.

### CODE REVIEW

**Comment by ylemkimon** [comment]
Another way to do this is to add `defer` to both KaTeX and mhchem.

**Comment by ronkok** [comment]
Good point. I'll modify the docs.

File: contrib/mhchem/README.md [hunk 1]
```diff
@@ -7,9 +7,11 @@ This extension adds to KaTeX the `\ce` and `\pu` functions from the [mhchem](htt
 This extension isn't part of core KaTeX, so the script should be separately included. Write the following line into the HTML page's `<head>`. Place it *after* the line that calls `katex.js`.
 
 ```html
-<script src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/mhchem.min.js" integrity="sha384-5gCAXJ0ZgozlShOzzT0OWArn7yCPGWVIvgo+BAd8NUKbCmulrJiQuCVR9cHlPHeG"></script>
+<script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/mhchem.min.js" integrity="sha384-5gCAXJ0ZgozlShOzzT0OWArn7yCPGWVIvgo+BAd8NUKbCmulrJiQuCVR9cHlPHeG"  crossorigin="anonymous"></script>
 ```
 
+If you remove the `defer` attribute from the `<script src="https://../katex.min.js">` tag, then you must also remove it from the `mhchem` tag. `mhchem.js` and `katex.js` must synchronize.
+
```
Comment (ylemkimon) [inline contrib/mhchem/README.md:14]: In theory, it's possible to only have `mchem` deferred.

Comment (ronkok) [reply contrib/mhchem/README.md]: Point taken. I've edited the sentence.

**Review by ylemkimon — APPROVED** [review]

**Pull request merged**